app/templates/layouts/base.html

```diff
@@ -8,29 +8,35 @@
         {% block custom_head_tags %}{% endblock %}
     </head>
     <body>
-      {# Example dropdown menu setup. Uncomment lines to view
-        {% set dropdown = 
-          [
-            ('account stuff',
-              [
-                ('account.login', 'login', 'sign in'),
-                ('account.logout', 'logout', 'sign out'),
-                ('2nd drop', [
-                  ('account.login', 'login 2', ''),
-                  ('3rd drop', [
-                    ('main.index', 'home 2', '')
-                  ])
-                ])
-              ]
-            ),
-            ('main.index', 'home 1', 'home')
-          ]
-        %}
-      #}
```

Comment:
I removed the sample dropdown provided in comments and added an actual drop-down for account options (editing profile, logging out, etc).